po4a.cfg

```
@@ -1,4 +1,4 @@
-[po4a_langs] pl
+[po4a_langs] es
```

```suggestion
[po4a_langs] pl es
```
Hold on, let's set up Spanish first. We have automation for that. Then you can translate.

---

ok